### ISSUE TITLE
Replace text in the client app with internationalizing formatter calls to expedite translation

### ISSUE THREAD

**Issue by ed-snodgrass** [Call-for-Code-for-Racial-Justice/Truth-Loop#242]
### Background on the problem the feature will solve/improved user experience
When users come to the application we want them to be able to read text in the language they're most comfortable. In order to do this we must replace all plain text displays with calls to the internationalization formatter. At the time of this issue being created that is `react-intl`. You may see examples in the client app by looking for `intl.formatMessage`

### Describe the solution you'd like
Display text should be moved to the `/src/nls/messages/en/messages.js` file and then properly looked up from there by the component.

### Tasks 
For each piece of text being moved
- [ ] Write a component test (or add an assertion to an existing test) to verify the text displays on the screen
- [ ] Move the text to the English translation file. Do not add to other translation files unless you can also provide that translation. Hint: Reuse things like 'Add', 'Update', etc...
- [ ] Use the formatting pattern in place to pull the text
- [ ] Make sure test still passes

### Acceptance Criteria 
All of the remaining display text (text the user sees) has been moved to the English translation file.
Tests exist for the items moved.

**Comment by ed-snodgrass** [comment]
@Fiewor 


An example of where it is used:
https://github.com/Call-for-Code-for-Racial-Justice/Truth-Loop/blob/main/client/src/privacyNotice/PrivacyNotice.js

English:
https://github.com/Call-for-Code-for-Racial-Justice/Truth-Loop/blob/main/client/src/nls/messages/en/messages.js

Here is an example of where you can start:
https://github.com/Call-for-Code-for-Racial-Justice/Truth-Loop/blob/main/client/src/PolicyNotice.js
basically replace `Policy Notice` with `{intl.formatMessage(messages.policyNotice)}`

**Comment by ed-snodgrass** [comment]
I say create a pull request and in the meantime I'll look for other spots.
Thank you!
